docs/guide-end-of-form.rst

```diff
@@ -0,0 +1,243 @@
+
+If you want to guarantee that each draft is finalized from the form end screen, you can disable :guilabel:`Finalize all drafts` from the :ref:`protected access control settings <admin-settings>`.
+
+Set an admin password
+~~~~~~~~~~~~~~~~~~~~~
+
+If your data collectors are likely to want to change some of the settings that are important for your project, you can :ref:`specify an admin password <admin-password>` that will be required to view and change protected settings. If you do this, you will generally want to also hide all :guilabel:`User Settings` that are important for your project.
+
+In many cases, the admin password will never need to be used: its purpose is only to lock down settings. In that case, it can be long and impossible to remember. In some cases, it may be necessary for someone in the field such as a supervisor to be able to change settings. In that case, it should be set to something relatively easy to communicate and enter.
+
+.. _guide-form-states-common-workflows:
+
+Customizations for common workflows
+------------------------------------
+
+Some questions to ask yourself as you design your workflow are:
+
+* Is it possible for a data collector to reach the end of the form but still have information to fill in?
+* What should happen if a data collector is interrupted while filling out a form?
+* How capable are data collectors of remembering a workflow?
+* How many times will data collectors repeat the same workflow?
+* How capable are data collectors of making independent decisions when faced with unexpected situations like an interview being interrupted?
+* How trusted are data collectors? Are they likely to want to "cheat" in some way to save time and/or effort?
+* What are the consequences of incorrect data being submitted? What are the next steps if that happens and is detected?
+
+As you answer these questions, you will get a clearer sense of what needs to happen when data collectors need to exit a form. This section includes some common workflow patterns and how to use the tools outlined above to support them.
+
+No edits allowed after leaving data collection subject
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In many workflows, it's important to guarantee that data is not changed after the data collector no longer has access to the data collection subject. For example, a nurse administering a vaccine should generally capture all data about that vaccination encounter while their patient is with them. They should not rely on their memory to fill in details after the encounter. To guarantee that data collectors have to fill out the form in one session:
+
+* Remove option to save draft from the form end screen and from within the form filling experience.
+* Hide the Drafts button from the landing screen.
+* (Generally) Turn on :guilabel:`Auto send`
+* (If data is highly sensitive or devices are not trusted) Turn on :guilabel:`Delete after send`
```

Comment:
Not sure why these three are so lispy. Remove the () and make them normal sentences.